config/props.js

```diff
@@ -5,14 +5,16 @@ const productionProps = {
   env: process.env.NODE_ENV,
   dbUrl: process.env.LIVE_DB_URL,
   apiKey: process.env.LIVE_API_KEY,
+  omdb: process.env.OMDB,
```

Comment:
should this not be blah blah

Reply:
Do you want me to rename a variable to blah_blah?